dev/view/groups.html

```diff
@@ -0,0 +1,9 @@
+<div class="tab-pane fade" id="tab-groups">
+    <!--include 'groups/navigation.html' -->
+    <div class="tab-content inner-tab-content">
+        <!--include 'groups/public.html' -->
+        <!--include 'groups/private.html' -->
+        <!--include 'groups/create.html' -->
+        <!--include 'groups/manage.html' -->
+    </div>
+</div>
```

Comment:
This whole file should be inside the template, `popup_dev.html`. Nested templates will make the build process more complicated. Please move the html of this file inside `popup_dev.html`.